CairoMakie/src/primitives.jl

```diff
@@ -857,7 +857,7 @@ function surface2mesh(xs::AbstractVector, ys::AbstractVector, zs::AbstractMatrix
         QuadFace(idxs[i, j], idxs[i+1, j], idxs[i+1, j+1], idxs[i, j+1])
         for j in 1:size(zs, 2)-1 for i in 1:size(zs, 1)-1
     ]
-    normal_mesh(ps, faces)
+    GeometryBasics.mesh(normal_mesh(ps, faces); uv = Vec2f)
```

Comment:
You should be able to use `GeometryBasics.uv_normal_mesh(GeometryBasics.Mesh(ps, faces))` ... I think it's a bit nicer, since it doesn't go two times through `mesh` that way.

Reply:
Just saw this...will try that out as well!  I went with the approach of manually creating the uv, since it allows us to clean up CairoMakie's uv mesh handling as well (it was inverting the 'v' portion of uv when sampling colors for some reason)

Reply:
Looks like that works and is a lot simpler :D so have used `GeometryBasics.uv_normal_mesh(...)` instead.